documentation/asciidoc/topics/proc_graceful_shutdown.adoc

```diff
@@ -0,0 +1,12 @@
+[id='shutting_down-{context}']
+= Shutting Down {brandname} Clusters
+Gracefully stop all {brandname} pods in the cluster so that permanent cache definitions and persisted data is available the next time pods start.
+
+.Procedure
+
+* Set the value of `replicas` to `0` and apply the custom resources.
++
+----
+spec:
+  replicas: 0
+----
```

Comment:
MORE INFO
The cluster must wake up from a shutdown with the same number of pods, so to restart the cluster replicas must be set to the number of pods that were running just before the shutdown

Reply:
gotcha. so if there were 6 nodes in the cluster before shutdown, you must set `replicas: 6` when you bring it back up.

what are the consequences when you don't do that? say 6 nodes. shutdown. restart the cluster with 8 nodes. there isn't an automatic rebalance?

Reply:
I don't know the details, maybe @tristantarrant can help on this